### PR TITLE
Foods

### PR DESCRIPTION
**Pull request by jtruong2**
Create and delete foods, validations for form, clearing forms and validations. Still needs refactoring.
@emcooper @case-eee

### CODE REVIEW

File: foods.html [hunk 1]
```diff
@@ -3,9 +3,24 @@
 <head>
   <meta charset="utf-8">
   <title>Quantified Self - Manage Foods</title>
-</head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script></head>
```
Comment (case-eee) [inline foods.html:6]: i'd add this to your package json and allow webpack to load this for you 👍

File: lib/foods/foods.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+$(document).ready(function() {
+  $(loadFoods)
```
Comment (case-eee) [inline lib/foods/foods.js]: is there a reason you're wrapping this in a jquery object? why not just execute the function?

File: lib/foods/foods.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        "</tr>"
+      )
+    })
+  })
+}
+
+function deleteFood(e) {
+  $.ajax({
+    type: 'DELETE',
+    url:'https://obscure-harbor-85447.herokuapp.com/api/v1/foods/' + e.target.id,
```
Comment (case-eee) [inline lib/foods/foods.js]: it looks like you're reusing the base of this url. can we extract it out and make it a constant so we can reuse it? 👍

File: lib/foods/foods.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    success: $(`tr.row${e.target.id}`).remove()
+  })
+}
+
+$('form').submit(function(e) {
```
Comment (case-eee) [inline lib/foods/foods.js]: what if there's more than one form on the page? selecting your form in this way may cause problems in the future. can we be more selective here?

File: lib/foods/foods.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    success: function(data){
+      $('.foods-table').append(
+        '<tr><td>'+ data.name + '</td>' +
+        '<td>' + data.calories + '</td>' +
+        `<td><input id=${data.id} type='button' class='delete-row' value='Delete'/></td>` +
+        "</tr>"
+      )
+
+    },
+    error: function(error){
```
Comment (case-eee) [inline lib/foods/foods.js]: i like how you're handling the specific errors here.. but could we make this a named function and break this down even further? there's a lot going on here!

**Review by case-eee — COMMENTED** [review]
good work @jtruong2 & @emcooper - let me know if you have questions on my comments!

File: foods.html (hunk 1 above)
Comment (jtruong2) [reply foods.html:6]: @case-eee how would I go about adding it to my package.json?

File: lib/foods/foods.js (hunk 2 above)
Comment (jtruong2) [reply lib/foods/foods.js]: Good call. I just changed it

File: lib/foods/foods.js (hunk 4 above)
Comment (jtruong2) [reply lib/foods/foods.js]: Okay so scope it down to .foods-table .row# should work

**Pull request merged**